layouts/community/ortho_4x12/guidoism/keymap.c

```diff
@@ -91,6 +177,14 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
       }
       return false;
       break;
+    case FUNCTION:
```

Comment:
You could just use `MO(_FUNCTION)` here.  

And add `#define FUNCTION MO(_FUNCTION)` at the top, too.

Reply:
Done.